pkg/api/docs/merge/merge_specs.go

```diff
@@ -98,6 +135,61 @@ func mergeSpecs(output string, sources ...string) error {
 	return nil
 }
 
+// getDashboardDefinitions converts the dashboard OpenAPI v3 to OpenAPI v2
+// and it returns it as spec.Schema
+func getDashboardDefinitions() (spec.Definitions, error) {
+	lib := thema.NewLibrary(cuecontext.New())
+	lin, _ := dashboard.Lineage(lib)
+	// Grab the 0.0 version. Or whichever one you want
+	sch := thema.SchemaP(lin, thema.SV(0, 0))
```

Comment:
With what's been merged since you started on this, there's now a strongly preferable pattern:
```suggestion
	reg, _ := registry.ProvideStatic()
	sch := reg.Dashboard().CurrentSchema()
```

This delegates responsibility for choosing the current version of the dashboard schema to the coremodel itself, where the current version is decided. For consistency across the grafana codebase, it's important that decision be made centrally.

And, by using the static registry, this code will be statically discoverable, making it easy to identify as a place where there's a dependency on the dashboard lineage.